src/features/klant/types.ts

```diff
@@ -1,6 +1,15 @@
-import type { Klant } from "@/stores/contactmoment";
-
 export type UpdateContactgegevensParams = Pick<
   Klant,
   "id" | "telefoonnummers" | "emails"
 >;
+
+export interface Klant {
+  id: string;
+  klantnummer: string;
+  voornaam: string;
+  voorvoegselAchternaam?: string;
+  achternaam: string;
+  telefoonnummers: { telefoonnummer: string }[];
+  emails: { email: string }[];
```

Comment:
Would the following be possible? I don't understand the extra object and key.

```
telefoonnummers: { telefoonnummer: string }[]; // current
telefoonnummers: string[]; // could we do this?

```

Reply:
zo is het in lijn met de api structuur. aanpassen heeft iets meer impact en past niet binnen de scope van deze story. dus ik laat het nu liever zo